### DISCUSSION TITLE
Detection without motion

### DISCUSSION THREAD

**Discussion by cbnathanael** [blakeblackshear/frigate#756] [Ask A Question]
I'm trying to figure out how to process detection when there's no motion. The goal is to help control light automations in Home Assistant based on human presence. We frequently nap on the couch, and I want my automations to know whether or not someone is still in the room, even if they're not moving.

Is there a way to request a detection check via API call or MQTT?

**Comment by blakeblackshear** [comment]
There isn't a way to request a detection. Maybe you could use zones to assume a person is still in the room if the event ends when they are in the middle of the room.

**Reply by cbnathanael** [reply]
I *think* I understand the idea... Configure zones around the room, and if the "core" zone event ends without a subsequent "edge" zone event (the person did not move from core to edge) the person is most likely still in the room. 

That seems like a lot of state management with a lot of margin for error.

**Reply by blakeblackshear** [reply]
I completely agree, but I don't think there is another way to accomplish your goal in the current version.
